sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateSuite.scala

```diff
@@ -1708,6 +1713,102 @@ class TransformWithStateSuite extends StateStoreMetricsTest
       }
     }
   }
+
+  test("transformWithState - verify that no metadata and schema logs are purged after" +
```

Comment:
lets add another test case for add/remove/add cases as well ?

Reply:
Oh we already have a test case for this here: `"transformWithState - verify that metadata and schema logs are purged"`